src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -181,6 +182,22 @@ trait BackendCall {
       Future.successful(ExecutionHash("", None))
   }
 
+  def pollingInitialInterval: FiniteDuration
+  def pollingMaxInterval: FiniteDuration
+  def pollingMultiplier: Double
+  def pollingInitialGap: FiniteDuration
```

Comment:
Hmm but what would they be ? If they are different for every backend - having defaults would be like choosing a "more important default backend" no ?